flow/designs/asap7/jpeg/autotuner.json

```diff
@@ -8,21 +8,21 @@
         ],
         "step": 0
     },
-    "CORE_UTILIZATION": {
+    "EQUIVALENCE_CHECK": {
         "type": "int",
         "minmax": [
-            30,
-            100
+            0,
+            0
         ],
-        "step": 1
+        "step": 0
     },
```

Comment:
Let's use the approach I mentioned in the other comment instead of adding this to the search space. Adding an entry here would increase the number of dimensions in the search space -- which is suboptimal.

Reply:
There is no need to auto tune the use of eqy as it doesn't improve the results, it only verifies correctness.  I suggest disabling it when autotuning.

Reply:
Thanks for the clarification @vvbandeira @maliberty. Updated accordingly